CASMPET-7265 add node to management-nodes-rollout to restart goss-servers on worker nodes (#5503)
Cray-HPE/docs-csm: operations/iuf/workflows/management_rollout.md

      $ncn --format json | jq -r ' .id+" "+.desiredConfig+" status="+.configurationStatus'; done
    ```

1. (`ncn-m001#`) If this is an upgrade to CSM 1.6.0, it is required to restart the 'goss-servers' service on all worker nodes as some Goss tests may not be included due to a timing issue. This issue is fixed in CSM 1.6.1.

    ```bash

>     worker_nodes=$(grep -oP "(ncn-w\d+)" /etc/hosts | sort -u | tr -t '\n' ',')

Reviewer: Even though `pdsh` doesn't seem to mind it, please strip off the trailing comma:
```
 worker_nodes=${worker_nodes%,}
```

Author: I have included this in my new changes, thanks.

Return to the procedure that was being followed for `management-nodes-rollout` to complete the next step, either [Management-nodes-rollout with CSM upgrade](#31-management-nodes-rollout-with-csm-upgrade) or
[Management-nodes-rollout without CSM upgrade](#32-management-nodes-rollout-without-csm-upgrade).

## 4. Restart `goss-servers` on all NCNs

> The `goss-servers` service needs to be restarted on all NCNs. This ensures the correct tests are run on each NCN. This is necessary due to a timing issue in the NCN's cloud-init which has been fixed in CSM 1.6.1.

Reviewer 2: Shouldn't it be "which is going to be fixed in CSM 1.6.1" ?

Reviewer 3: We aren't going to want to make another PR to change the verb tense after 1.6.1 is released. But we do probably want to start these paragraphs with something that indicates it only applies if they are on a version lower than 1.6.1. I know we say it at the end, but it's nice to also mention up front, IMO.

```suggestion
## 4. Restart `goss-servers` on all NCNs

> Skip this step if the CSM version is 1.6.1 or above. This step will cause no harm if done on CSM 1.6.1 or higher, but it is unnecessary.

If the CSM version is 1.6.0 or lower, then the `goss-servers` service needs to be restarted on all NCNs. This ensures the correct tests are run on each NCN. This is necessary due to a timing issue that is fixed in CSM 1.6.1.
```

Author: I am going to leave this wording "which has been fixed in CSM 1.6.1" so that we do not need to change the verb tense with another PR later.

(`ncn-m001#`) Restart `goss-servers`.

>   ```bash
>   ncn_nodes=$(grep -oP "(ncn-s\w+|ncn-m\w+|ncn-w\w+)" /etc/hosts | sort -u | tr -t '\n' ',')
>   ncn_nodes=${ncn_nodes%,}
>   pdsh -S -b -w $ncn_nodes 'systemctl restart goss-servers'
>   ```

Reviewer 3: I don't think this should be indented.

````suggestion
```bash
ncn_nodes=$(grep -oP "(ncn-s\w+|ncn-m\w+|ncn-w\w+)" /etc/hosts | sort -u | tr -t '\n' ',')
ncn_nodes=${ncn_nodes%,}
pdsh -S -b -w $ncn_nodes 'systemctl restart goss-servers'
```
````

Author: Fixed

Return to the procedure that was being followed for `management-nodes-rollout` to complete the next step, either [Management-nodes-rollout with CSM upgrade](#31-management-nodes-rollout-with-csm-upgrade) or
[Management-nodes-rollout without CSM upgrade](#32-management-nodes-rollout-without-csm-upgrade).

## 4. Restart `goss-servers` on all NCNs

> > Skip this step if the CSM version is 1.6.1 or above. This step will cause no harm if done on CSM 1.6.1 or higher, but it is unnecessary.

Reviewer 3: This is more consistent with the preface to the next section, I see. Not a big deal, but as long as we're here.

```suggestion
**`NOTE`** Skip this step if the CSM version is 1.6.1 or above. This step will cause no harm if done on CSM 1.6.1 or higher, but it is unnecessary.
```
